spec/helium/client/sensors_spec.rb

```diff
@@ -39,9 +43,13 @@
     expect(sensor.mac).to eq("6081f9fffe000478")
   end
 
+  it 'has a firmware version' do
+    expect(sensor.firmware).to eq("4020100")
+  end
+
   it 'has ports' do
     # TODO Port will be an object
-    expect(sensor.ports).to eq(["_se","lr","l","p","h","m","_e.info","t","b"])
+    expect(sensor.ports.sort).to eq(["_se","lr","l","p","h","m","_e.info","t","b"].sort)
```

Comment:
nice, that works.

i was doing the following: 

```ruby
it 'returns all ports' do
      all_ports = ['t', 'h', 'p', 'l', 'b', '_se', 'm']
      expect(data_points.collect(&:port).uniq).to contain_exactly(*all_ports)
end
```

but sorting + eq works as well